main.js

```diff
@@ -0,0 +1,306 @@
+var query = null;
+var todayFetch = function(query) {
+  lastQuery = encodeURIComponent(query);
+  $.ajax({
+    method: "GET",
+    url: "https://api.openweathermap.org/data/2.5/weather?q=" + lastQuery + "&APPID=3539d3ddceaed0d39c41bd28668ccd0e",
+    dataType: "json",
+    beforeSend: function() {
+      $('#loaderDiv').show();
+    },
+    success: function(data) {
+      $('#loaderDiv').hide();
+      addToday(data);
+      console.log("today weather Fetch Successful");
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      $('#loaderDiv').hide();
+      alert("NOT A VALID CITY");
+      console.log(textStatus);
+      console.log('FAIL!');
+    }
+  })
+}
+
+var dateStrToName = function(dateStr) {
```

Comment:
Nice! You might want to name this `_ dateStrToName` to help identify it as a utility function